### ISSUE TITLE
Group elements in CRUD lists, keep everything working

### ISSUE THREAD

**Issue by Floby** [centurion-project/Centurion#68]
As of now, you can easily override getSelectFiltred in your CRUD controllers to group some rows together (even though you have to reset the group statement of the request). I did it to group media by their sha1 property.

However this does break some functionnalities:
- _Actions by lot_ : checking a few boxes and calling an action on them only affects the first row of a group.
- _individual action_ : clicking on an action link only affects the first row of a group
- Basically anything that expects only one tuple of primary key. (usually just the `id` property).

I already gave some thoughts to it and I couldn't find a way to do this without modifying huge parts of the code in many different places. So I'm filing this feature request instead of trying to make it a pull request because it still would be nice to have.

**Comment by takarboost** [comment]
Hi ,
I would like to group by some table fields, but i dont know how
could you help me please
